README.md

```diff
@@ -2,528 +2,92 @@
-```
-
-The `Management` class stores access to endpoints as properties of its instances. The best way to see what endpoints are covered is to read through the `\Auth0\SDK\API\Management::__construct()` method. 
-
-### Example - Search Users by Email
-
-This endpoint is documented [here](https://auth0.com/docs/api/management/v2#!/Users/get_users).
-
-```php
-$results = $mgmt_api->users->search([
-    'q' => 'josh'
-]);
-
-if (! empty($results)) {
-    echo '<h2>User Search</h2>';
-    foreach ($results as $datum) {
-        printf(
-            '<p><strong>%s</strong> &lt;%s&gt; - %s</p>',
-            !empty($datum['nickname']) ? $datum['nickname'] : 'No nickname',
-            !empty($datum['email']) ? $datum['email'] : 'No email',
-            $datum['user_id']
-        );
-    }
-}
-```
-
-### Example - Get All Clients
-
-This endpoint is documented [here](https://auth0.com/docs/api/management/v2#!/Clients/get_clients).
-
-```php
-$results = $mgmt_api->clients->getAll();
-
-if (! empty($results)) {
-    echo '<h2>Get All Clients</h2>';
-    foreach ($results as $datum) {
-        printf(
-            '<p><strong>%s</strong> - %s</p>',
-            $datum['name'],
-            $datum['client_id']
-        );
-    }
-}
-```
+The values above should not be hard-coded in a production application but will suffice for testing or local development. Please see our complete guide on the [main documentation page](https://auth0.com/docs/libraries/auth0-php#getting-started) for more information on how to store and use these values.
 
 ## Contributing
 
-We provide and maintain SDKs for the benefit of our developer community. Feedback, detailed bug reports, and focused PRs are appreciated. Thank you in advance!
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
 
-When contributing to this SDK, please:
+- [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+- [This repo's contribution guide](CONTRIBUTING.md)
 
-- Maintain the minimum PHP version (found under `require.php` in `composer.json`).
-- Code to the [PSR-2 standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md).
-- Write tests and run them with `composer test`.
-- Keep PRs focused and change the minimum number of lines to achieve your goal.
+## Support + Feedback
 
-To run tests on the SDK, you'll need to create a `.env` file in the root of this package with the following entries:
+- Use [Community](https://community.auth0.com/) for usage, questions, specific cases
+- Use [Issues](https://github.com/auth0/auth0-PHP/issues) here for code-level support
+- Paid customers can use [Support](https://support.auth0.com/) to submit a trouble ticket for production-affecting issues. 
```

Comment:
I'd normally merge both "community" and "support" under `https://support.auth0.com/` which will route them accordingly.

Reply:
I think it's good to call them both out.